### DISCUSSION TITLE
Disable transcoding of uploaded videos

### DISCUSSION THREAD

**Discussion by cinetube** [mediacms-io/mediacms#654] [Feature Requests]
I want to disable transcoding of uploaded videos. I want all the videos can only be viewed at their original resolution. 

How to implement it?

I am planning to use Mediacms to host my home videos on Raspberry Pi.

**Comment by tschig** [comment]
Did you check https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#12-video-transcoding ?

**Comment by cinetube** [comment]
> Did you check https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#12-video-transcoding ?

I checked that. But MediaCMS doesn't play uploaded videos until the video is transcoded.

**Comment by mgogoulos** [comment]
Hi, sorry for replying late to this, so there has been a branch that allows this, you have to checkout this specific branch. This could also be brought to main branch with a little more testing.. - https://github.com/mediacms-io/mediacms/discussions/285
